src/toast/todmap/pointing.py

```diff
@@ -117,6 +123,16 @@ def mode(self):
         """
         return self._mode
 
+    @property
+    def local_submaps(self):
+        """(list): Indices of locally hit submaps
+        """
+        if self._single_precision:
+            dtype = np.int32
+        else:
+            dtype = np.int64
+        return np.arange(self._nsubmap, dtype=dtype)[self._hit_submaps]
```

Comment:
Even at NSIDE=8192, with the default nside_submap=16, the length of the np.arange (before slicing, with int64) would still only be a couple of MB, which is fine.